### ISSUE TITLE
mist_device_gateway : Provider produced inconsistent result after apply `.additional_config_cmds`

### ISSUE THREAD

**Issue by zbuchheit** [Juniper/terraform-provider-mist#27]
When using the following hcl
```tf
terraform {
  required_providers {
    mist = {
      source  = "juniper/mist"
      version = "0.1.33"
    }
  }
}

resource "mist_device_gateway" "gateway_one" {
  name        = "gateway_one"
  device_id = "redacted"
  site_id   = "redacted"
  oob_ip_config = {
    type = "dhcp"
  }
  additional_config_cmds = []
  dns_servers = ["8.8.8.8"]
  managed = false
}
```

I get the following error during `terraform apply`
```
mist_device_gateway.gateway_one: Creating...
╷
│ Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to mist_device_gateway.gateway_one, provider
│ "provider[\"registry.terraform.io/juniper/mist\"]" produced an unexpected new value:
│ .additional_config_cmds: was cty.ListValEmpty(cty.String), but now null.
│ 
│ This is a bug in the provider, which should be reported in the provider's own issue
│ tracker.
```

**Comment by tmunzer** [comment]
fixed in v0.1.34, I added  `listvalidator.SizeAtLeast(1)` to all `additional_config_cmds` (but I just saw I forgot some of them... I'll fix that in the next release)

**Comment by zbuchheit** [comment]
it looks fixed in the latest release from my testing
